rfcs/proposed/iterators_pass_directly_customization/README.md

```diff
@@ -0,0 +1,170 @@
+# Passed Directly Customization Point for User Defined Types
+
+## Introduction
+
+oneDPL handles some types of input data automatically as input to its device backend (SYCL-based) as described
```

Comment:
If I were a user (not oneDPL developer) I would had not understood what phrase "handles some types of input data automatically as input to its device backend" mean... And also I would had not understood what is "passed directly" and why it might be helpful.
I think we have to  explain it more clear by saying about copying data (under an iterator) between the host and device in implicit way in case of "passed directly" is false.

Reply:
Take a look and see if the changes help clarify this section.